node/src/main/scala/org/bitcoins/node/config/NodeAppConfig.scala

```diff
@@ -106,6 +106,13 @@ case class NodeAppConfig(
     }
   }
 
+  lazy val relay: Boolean = {
+    if (config.hasPath("bitcoin-s.node.relay"))(config.getBoolean(
```

Comment:
let's clean up the format here. Scala gives you a lot of optionality with syntax, but it is 99% of the time best to use braces. 

```scala
if (config.hasPath("bitcoin-s.node.relay")) {
  config.getBoolean("bitcoin-s.node.relay")
} else {
  false
}